### ISSUE TITLE
Considered property for Review Assignments is not correctly saved

### ISSUE THREAD

**Issue by jardakotesovec** [pkp/pkp-lib#9751]
**Describe the bug**
When editor clicks on Read Review. It should save 'considered = 3' to the database, but it saves considered = 1. So it does not change to 'Thank to reviewer' button and probably causes some other issues.

**To Reproduce**
1. Editor Assign Reviewer
2. Reviewer confirms and submits the review
3. Editor clicks on 'Read Review'

**What application are you using?**
OJS 3.5

**Additional information**
This is caused with newly added validation schema **reviewAssignment.json**, which is incorrect
```
    "considered": {
            "type": "boolean",
            "description": "Whether reviewer considers a review assignment request",
            "apiSummary": true,
            "validation": [
                "nullable"
            ]
        },
```

As result when saving considered = 3, validation silently changes that to considered = 1.

Maybe validation in such case should silently fail rather than silently break it?

**Comment by touhidurabir** [comment]
Yes, that validation schema is causing this as `considered` value can be in range from `0` to `3` , see at https://github.com/touhidurabir/pkp-lib/blob/main/classes/submission/reviewAssignment/ReviewAssignment.php#L40-L44 . The validation schema of `reviewAssignment.json` for `considered` should be like 
```
"considered": {
            "type": "integer",
            "description": "Whether reviewer considers a review assignment request",
            "apiSummary": true,
            "validation": [
                "nullable",
                "int:0,1,2,3"
            ]
        },
```

**Comment by Vitaliy-1** [comment]
PR pkp-lib: https://github.com/pkp/pkp-lib/pull/9770
tests: https://github.com/pkp/ojs/pull/4204
